src/clusterfuzz/_internal/datastore/data_types.py

```diff
@@ -423,6 +423,9 @@ class Testcase(Model):
   # Timestamp.
   timestamp = ndb.DateTimeProperty()
 
+  # Original creation timestamp.
+  creation_timestamp = ndb.DateTimeProperty()
```

Comment:
nit: add `indexed=False` here, unless we plan on querying on this property. 

Also, I'd rename this to `created`